platform-hub-web/src/app/kubernetes-tokens/kubernetes-tokens-form.component.js

```diff
@@ -30,6 +30,10 @@ function KubernetesTokensFormController($state, hubApiService, logger, _, Kubern
     ctrl.isNew = !cluster;
     ctrl.loading = true;
 
+    if (!FeatureFlags.isEnabled(FeatureFlags.keys.kubernetesTokens)) {
+      $state.go('home'); // feature disabled
+    }
```

Comment:
(The problem with doing this here is it's not guaranteed that the feature flags have been fetched from the server at this point).